lib/wrappers/transform-node.ts

```diff
@@ -18,7 +17,13 @@ export default class TransformNodeWrapper extends NodeWrapper {
       outputPath: this.outputPath,
       cachePath: this.cachePath,
     });
+    //adding ts-ignore for now. Should be removed later when new version of broccoli-node-api is released
```

Comment:
Can you make an issue to clean this up? I really would prefer to not have to keep these ts-ignores for too long...

Reply:
I have tasks mentioned in the PR description. Once we are ready to Merge this PR. we will bump the `broccoli-node-api` and `broccoli-node-info` then merge this change.